app/src/main/java/org/simple/clinic/sync/SyncScheduler.kt

```diff
@@ -21,7 +21,7 @@ class SyncScheduler @Inject constructor(
   fun schedule(): Completable {
     return Single.just(syncConfig)
         .map { config -> createWorkRequest(config.syncInterval) }
-        .doOnSuccess { request -> workManager.enqueueUniquePeriodicWork("sync-patient-resources", REPLACE, request) }
+        .doOnSuccess { request -> workManager.enqueueUniquePeriodicWork(syncConfig.name, REPLACE, request) }
```

Comment:
If we are going to use the `SyncConfig#name` as only for the unique work name, can't we just a use a constant defined in `SyncConfig` instead of as a class param 🤔

Reply:
We can, but we shouldn't. Each `SyncConfig` is mapped to one kind of unique work, each of which requires a name. So I made it a property.